code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/floatingbutton/views/FloatingButton.kt

```diff
@@ -58,8 +58,8 @@ internal fun FloatingButton(
     onDragFinished: (Offset) -> Unit
 ) {
     // Floating button draggable area dimensions
-    val heightDp = with(LocalConfiguration.current) { remember { mutableStateOf(screenHeightDp.dp) } }
-    val widthDp = with(LocalConfiguration.current) { remember { mutableStateOf(screenWidthDp.dp) } }
```

Comment:
Do we also need to recompute the offset for the floating when the orientation is changed?